migrations/create_db.sql

```diff
@@ -1,5 +1,10 @@
 DROP DATABASE IF EXISTS spark;
 
+# Implicitly create the user and then drop the user.
+GRANT USAGE ON *.* TO 'spark'@'localhost' IDENTIFIED BY 'password';
+DROP USER 'spark'@'localhost';
+flush privileges;
```

Comment:
Good point, we should discuss it. My code change doesn't change the behavior of user creation, it simply address an issue that might rise when running the same script more than once.
In the meantime, I want to commit to resolve issues on master.